contrib/scripts/make_user_admin_in_room.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
```

Comment:
Adding a small description of what this script does and the minimum-required Python version up at the top here would be helpful.